apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -331,6 +347,21 @@ loginView { state, setState } = React.fragment
             ]
         ]
 
+errorAlert :: Error -> Maybe Alert
+errorAlert err = oneOf
+  [ do { method, url } <- Persona.networkError err
+       pure
+         { level: Alert.danger
+         , title: "Failed to connect."
+         , message: "Check your connection and try again later."
```

Comment:
I think we were waiting for the translated Swedish texts, should we have them in this PR or in a new one?